netlink/netlink_windows.go

```diff
@@ -2,3 +2,82 @@
 // MIT License
 
 package netlink
+
+import "net"
+
+// Link represents a network interface.
+type Link interface {
+	Info() *LinkInfo
+}
+
+type Route struct{}
+
+// LinkInfo respresents the common properties of all network interfaces.
+type LinkInfo struct {
+	Type string
+	Name string
+}
+
+func (linkInfo *LinkInfo) Info() *LinkInfo {
+	return linkInfo
+}
+
+func (Netlink) AddLink(link Link) error {
+	return nil
```

Comment:
that this whole file is a stub in the os-specific build makes me think that this abstraction is wrong